packages/app/src/components/nominationBanner/nominationBanner.js

```diff
@@ -69,8 +69,10 @@ const NominationBanner = (props) => {
     );
   };
   function hasAttachments() {
-    setActiveNomination({...activeNomination, attachments: false})
-    return resetAttachments(activeNomination)
+    let newNomination = activeNomination;
+    newNomination.attachments = false;
```

Comment:
I think you'll need to update it this way for React to update the UI. React checks if the two objects are equal to see if it needs to rerender and without the spread operator that creates a new object I think it won't detect the update.
```suggestion
    let newNomination = {...activeNomination, attachments: false}`;
```